contrib/experimental/great_expectations_experimental/expectations/expect_column_values_to_be_present_in_other_table.py

```diff
@@ -0,0 +1,299 @@
+from typing import List, Optional, Union
```

Comment:
Could we add `from __future__ import annotations` and `if TYPE_CHECKING`?

Reply:
done!